### PR TITLE
[NUX-640] - New Date Time Kit

### PR DESCRIPTION
**Pull request by jasoncypret**
#### Screens
date-time

date-time-code

#### Breaking Changes
I believe this is not a breaking change. (Still investigating).

#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/NUX-640

#### How to test this
Kit available in Playbook.

#### Checklist:

- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [x] **SCREENSHOT** Please add a screen shot or two
- [x] **SPECS** Please cover your changes with specs

### CODE REVIEW

File: app/pb_kits/playbook/pb_date/_date.jsx [hunk 1]
```diff
@@ -28,9 +29,10 @@ const PbDate = (props: PbDateProps) => {
     id,
     showDayOfWeek = false,
     showIcon = false,
+    timeZone,
   } = props
 
-  const dateTimestamp = new DateTime({ value: date })
+  const dateTimestamp = new DateTime({ value: date, zone: timeZone })
```
Comment (kre8sions) [inline app/pb_kits/playbook/pb_date/_date.jsx:35]: @jasoncypret is this the spot you mentioned about multiple timezones being included?

cc:@coleerikson

Comment (coleerikson) [reply app/pb_kits/playbook/pb_date/_date.jsx:35]: I think the one @jasoncypret mentioned is here: https://github.com/powerhome/playbook/pull/1080

**Comment by jasoncypret** [comment]
@coleerikson Whats the work on this one?

**Comment by coleerikson** [comment]
This needs an update but it can go in next week for sure.

**Review by bh247484 — APPROVED** [review]

**Pull request merged**